apollo-portal/src/main/resources/static/i18n/en.json

```diff
@@ -569,6 +569,18 @@
   "UserMange.Email": "Email",
   "UserMange.Created": "Create user successfully",
   "UserMange.CreateFailed": "Failed to create user",
+  "UserMange.Enabled": "Enabled",
```

Comment:
should we sort it by alphabeta?

Reply:
It would be best to sort the labels by alphabet order, but it's also acceptable if not sorted.